CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/state/AnalyticViewState.java

```diff
@@ -62,8 +59,7 @@ public AnalyticViewState(final AnalyticViewState state) {
         this.currentQuestion = state.getCurrentQuestion();
         this.question = state.getQuestion();
         this.activeCategory = state.getActiveCategory();
-        this.graphVisualisations = state.getGraphVisualisations();
-        this.internalVisualisations = state.getInternalVisualisations();
+        this.graphVisualisations = (HashMap) state.getGraphVisualisations();
```

Comment:
HashMap cast shouldn't be needed

Reply:
The casts to HashMap in the state class is required in order to be able to use .clone() in the getter.

Reply:
right. I've just realised that `graphVisualisations` is declared as `HashMap` (missed that in my last review). If you change that declaration to `Map`, the cast shouldn't be necessary

Reply:
The .clone() method isn't available in the Map class which is why this has to be declared as a HashMap and all of the casts that are currently left need to stay. I have removed all other casts that are not necessary.

Reply:
Gotcha. Didn't realise `clone` wasn't available